platform/jobs/pipelines/edxPlatformLettucePipelinePr.groovy

```diff
@@ -2,7 +2,6 @@ package platform
 
 import org.yaml.snakeyaml.Yaml
 import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
-import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_TEAM_SECURITY
```

Comment:
I know this was copy/paste, but I am just curious what your thoughts are for private pipeline jobs.We can talk about that in slack if you want

Reply:
Perhaps we should add a separate ticket to incorporate private jobs

Reply:
Sounds good